src/components/Calculator.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] `Required`
- [ ] Please enable the disabled eslint because it defeats the purpose of having linters in the project. 🥂